File: DotNetCore/src/DevIO.Business/Models/Cliente.cs
using System;
using System.Collections.Generic;

namespace DevIO.Business.Models
{
    public class Cliente : Entity
    {
        public string Nome { get; set; }
        public DateTime DataNascimento { get; set; }
        public string NomeMae { get; set; }
        public bool Ativo { get; set; }
        public Guid EstadoCivilId { get; set; }
        public EstadoCivil EstadoCivil { get; set; }
        public IEnumerable<Endereco> Enderecos { get; set; }
    }
}

File: Angular/src/app/cliente/models/cliente.ts
import { EstadoCivil } from './EstadoCivil';
import { Endereco } from 'src/app/endereco/models/endereco';

export class Cliente {
    id: string;
    nome: string;
    nascimento: Date;
    estadoCivil: EstadoCivil;
    nomeMae: string;
    ativo: boolean;
    enderecos: Endereco[];
}